src/mac_features/ack_generator/nrf_802154_ack_data.h

```diff
@@ -99,6 +101,19 @@ bool nrf_802154_ack_data_for_addr_clear(const uint8_t * p_addr, bool extended, u
  */
 void nrf_802154_ack_data_reset(bool extended, uint8_t data_type);
 
+/**
+ * @brief Select the source matching algorithm.
+ *
+ * @note This method should be called after driver initialization, but before transceiver is enabled.
+ *
+ * When calling @ref nrf_802154_ack_data_pending_bit_should_be_set method, it will choose one of
```

Comment:
```suggestion
 * When calling @ref nrf_802154_ack_data_pending_bit_should_be_set, this function will choose one of
```
`it` refers to this function, correct?

Reply:
Sounds ok.